README.md

```diff
@@ -1,7 +1,252 @@
 # Fediscoverer - Fediverse Search and Discovery Provider 
 
-Heavy work in progress. See [Fediscovery website](https://fediscovery.org) for
-background information.
+fediscoverer is a Fediverse Discovery Provider allowing fediverse
+servers to share publicly discoverable accounts and content to improve
+search and discovery on individual servers.
+
+See [Fediscovery website](https://fediscovery.org) for background information.
+
+This project is in very early stages but we appreciate feedback.
+
+## Installation
+
+### Manual
+
+Prerequisites: 
+
+* Ruby (we recommend using a version manager such as mise, asdf or
+  rbenv, which will pick up the correct version from `.ruby-version`)
+* PostgreSQL (tested with version 16, but others should work just as
+  well)
+
+Note that the following is a high-level description of the installation
+steps. Some details may vary depending on the operating system or distro
+used.
+
+#### Step 1: Clone the Repo:
+
+```sh
+git clone https://github.com/mastodon/fediscoverer.git
+cd fediscoverer
+```
+
+#### Step 2: Install Ruby Dependencies
+
+```sh
+bundle install
+```
+
+(Note that installation of Ruby gems may fail if they depend on
+some native library and development headers being present. One such
+example could be the PostgreSQL client library. In this case you need to
+install the missing libraries using your operating system's package
+manager.)
+
+#### Step 3: Create a Secret Key Base
+
+Running
+
+```sh
+bin/rails secret
+```
+
+will print out a random secret suitable for use as Rails' "secret key
+base", a value used as a basis for cryptographic operations that should
+be unique to every installation of fediscoverer.
+
+Place this value in the `SECRET_KEY_BASE` environment variable so it is
+available in the following steps:
+
+```sh
+export SECRET_KEY_BASE=<long random key here>
+```
+
+Also make note of this value for later use. You should also consider
+adding this to the shell startup scripts of the user running
+fediscoverer or a `.env` file or similar, so you do not need to repeat
```

Comment:
We should probably document using `.env.production` here, as it's supported?

And mention that it can also go into their unit file if they use systemd? (but should we recommend writing it directly into a unit file? I am not sure)